main-server.js

```diff
@@ -28,7 +30,9 @@ const defaultOptions = {
 
 export const createApolloServer = (givenOptions, givenConfig) => {
 
+  let graphiqlOptions = Object.assign({}, defaultConfig.graphiqlOptions, givenConfig.graphiqlOptions);
   let config = Object.assign({}, defaultConfig, givenConfig);
```

Comment:
I think this line should take care of all attributes, including `graphiqlOptions`?